client/src/components/SideInfoPanal/SideInfoPanal.jsx

```diff
@@ -8,7 +8,6 @@ import GamesContext from '../../context';
 
```

Comment:
Panal should be Panel. If you Cmd+Shift+F you can search for every instance of Panal in the entire working directory, and then easily replace them all. If you highlight one instance of Panal on a page, you can hit Cmd+D until you have them all selected, then change them all at once. Shouldn't be more than a few minutes to change the entire project.

Reply:
Haha yeah, looks like I spelled it like that the whole time. Thanks.